src/Apps/Artist/Routes/Consign/Utils/getConsignmentData.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Passed https://docs.google.com/spreadsheets/d/1EicHOCyCozsPBkVOkQGTTUtEds7qQuacUAgLy-C2-Nk/edit#gid=0
+ * through a CSVtoJSON converter
+ */
+
+import { flatten, groupBy } from "lodash"
+import { staticCSVToJSONData } from "./staticCSVToJSONData"
+
+export interface ArtistConsignment {
+  artworks: object[]
+  metadata: {
+    highestRealized: string
+    realized: string
+    recentlySoldArtworkIDs: string[]
+    roundedUniqueVisitors: string
+    roundedViews: string
+    str: string
+    uniqueVisitors: string
+    views: string
+  }
+}
+
+/**
+ * Massage data to be more usable
+ */
+export function getConsignmentData(pathname: string): ArtistConsignment {
```

Comment:
If I'm reading this right, I believe we're calling this function twice on load - once in the route file and once in the component, is that right? Any perf concerns there? Sorta thinking out loud if there would be any advantage to caching anything here. 🤔

Reply:
It's probably ok for now, but we can keep that in mind.